components/brave_ads/common/features.cc

```diff
@@ -62,6 +56,12 @@ constexpr char kFieldTrialParameterNotificationAdDarkModeBackgroundColor[] =
 // Default color value is SkColorSetRGB(0x20, 0x23, 0x27);
 constexpr char kDefaultNotificationAdDarkModeBackgroundColor[] = "202327";
 
+// Set to true to support multiple displays or false to only support the primary
+// display
```

Comment:
nit: `.` at end of sentence

Reply:
All other comments in this file don't have `.` at the end of a sentence. Maybe it is better to fix all of them as a followup?

Reply:
Sounds good to me

Reply:
I will raise a PR over the weekend as in the process of adding some public facing comments anyhow. Thanks